ignite/pkg/cosmosclient/cosmosclient_test.go

```diff
@@ -0,0 +1,378 @@
+package cosmosclient
```

Comment:
```suggestion
package cosmosclient_test
```

to make sure that we're making black box testing